strings/src/main/res/values/strings.xml

```diff
@@ -1217,4 +1218,16 @@
     <string name="close_snackbar">Close snackbar</string>
 
     <string name="edit_finalized_form_warning">In later releases, you will not be able to edit finalized forms. Save forms as draft to edit them later.\n\nYou can check for errors in a draft form by tapping the three dots (⋮) and then Check for errors.</string>
+
+    <!-- Label for option that finalizes all the forms the user is currently looking at -->
+    <string name="finalize_all_forms">Finalize all forms</string>
+
+    <!-- Message shown when finalizing all forms has succeeded and how many totally have been finalized -->
+    <string name="bulk_finalize_success">Success! %d forms finalized.</string>
+
+    <!-- Message shown when no forms finalized successfully -->
+    <string name="bulk_finalize_failure">%d forms have errors. Address issues before finalizing all forms.</string>
```

Comment:
Nit: should "issues" be "errors"? My brain seems to want to make them two different concepts when there are two words. It might make localization harder, too.

Reply:
@alyblenkin what do you think?